multiuser_service_windows.go

```diff
@@ -282,6 +284,8 @@ func removeService(name string) error {
 		return fmt.Errorf("service %s is not installed", name)
 	}
 	defer s.Close()
+	// throw away this error
```

Comment:
Can you add a note to say why we ignore the error?

Reply:
If this fails it's essentially because something went wrong interacting with the service, i.e. it's not running, it didn't respond, etc. We don't care about an error in this scenario, we're just trying to delete it.

Reply:
Perfect. Could you include that as a code comment?